sdk/src/types/block/output/mod.rs

```diff
@@ -75,17 +78,27 @@ pub enum OutputBuilderAmount {
     MinimumAmount(StorageScoreParameters),
 }
 
-/// Contains the generic [`Output`] with associated [`OutputMetadata`].
+/// Contains the generic [`Output`] with associated [`OutputIdProof`] and [`OutputMetadata`].
 #[derive(Clone, Debug)]
+#[cfg_attr(
+    feature = "serde",
+    derive(serde::Serialize, serde::Deserialize),
+    serde(rename_all = "camelCase")
+)]
 pub struct OutputWithMetadata {
-    pub(crate) output: Output,
-    pub(crate) metadata: OutputMetadata,
+    pub output: Output,
+    pub output_id_proof: OutputIdProof,
+    pub metadata: OutputMetadata,
```

Comment:
`OutputWithProofAndMetadata`? :trollface:

Reply:
out of interest: why couldn't we put the proof inside of the metadata from a TIP perspective?

Reply:
One could have also structured it in a different way, but it made more sense this way because the output and the proof is static data and the metadata is dynamic, also the proof can only be used together with the output, having it in the metadata without the output itself doesn't help (the proof is also returned in the route with the output alone)